.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -24,7 +24,7 @@ The following checks can be performed after we have forked off to the release-ca
 
 - [ ] Verify the new migrations complete successfully, and the runtime state is correctly updated for any public (non-private/test) networks.
 - [ ] Verify the SDK is up-to-date with the latest changes.
-- [ ] Push runtime upgrade to local clone of Battery Station and ensure the upgrade is executed without errors.
+- [ ] ~~Push runtime upgrade to local clone of Battery Station and ensure the upgrade is executed without errors.~~ Removed since integration tests do the upgrade tests
```

Comment:
We should replace this with "Run integration tests." then. The way you've changed this, we now still have a box we have to tick every time we go through the checklist.

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/1332/commits/30701e13640b82c5b0ef75feb47cc83c4e4b5c23